### DISCUSSION TITLE
Welcome to VisiCut Discussions!

### DISCUSSION THREAD

**Discussion by t-oster** [t-oster/VisiCut#595] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**What is the difference to issues?**
We should use discussions to share ideas or get help and use issues if there is really a bug. If you just want to propose/request a new feature, we should start with discussions, unless you already provide part of the solution, then you should make a pull request.

**Comment by D4ss** [comment]
I have been running Lightburn on Windows but I am considering switching to Visicut on a RPi ( 4 or 5).  Are there any issues/snags I should be aware of? Thanx in advance.
